js/game.js

```diff
@@ -0,0 +1,10 @@
+class Game {/**Evil comments with my freaky peasant language skills*/
+    constructor() { 
+      this.gameRound = this.gameRound /**This is like a status, how many times has been played, gamerond act as iterator */
+    }
+    this.startgame  = startgame() /**Start new game as new player with new full decks and zero points */
+    this.stay = stay() /**If points achieve level nearly 21 or equal to 21, then player can pass move, withot taking a card*/
+    this.continuegame = continuegame() /**When winner is clear and loser want to win, then game continue with cards that left in deck*/
+    this.quitgame = quitgame() /**If player want to avoid addiction to gambling, then game can be ended, by exiting from it */
```

Comment:
Šīs metodes neko nedod, beidzas ar kļūdu. Metodes, mainīgos, klašu nosaukumus cenšamies rakstīt camelCase, citās valodās mainīgos/metodes raksta snake_case. Metodes uzrakstītas sintaktiski nepareizi.
ES6 sintaksē tukša metode, piemēram:
```js
this.startGame = () => ()
```
Lai gan neredzu jēgu definēt tukšu metodi. Ja tā ir metode kā mēģināt saprast/plānot risinājuma arhitektūru, tad tas laikam ir labāk nekā nekas.

Reply:
Jā, to tukšo metožu kopums ir mēģinājums saprast un plānot spēles risinājuma arhitektūru. Paldies par camelCase atgādinādinājumu, tā ir mana kļūda.